media/css/mozorg/home/fundraiser2018.scss

```diff
@@ -0,0 +1,122 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+$font-path: '/media/fonts';
+$image-path: '/media/protocol/img';
+
+@import '../../../protocol/css/includes/lib';
+@import '../../../protocol/css/components/button';
```

Comment:
The `button` component is now included in the base protocol style sheet for bedrock, so there should be no need to import it again here.

Reply:
I think it's needed for the old Pebbles-based home page. So it's a bit redundant to include here, but probably less redundant than making a whole separate Pebbles version of the style sheet. I do cringe a bit mixing Protocol and Pebbles on the same page but at least this banner will be short-lived.

Reply:
Ah I see, that makes sense. The only downside to this is it leaves a fair chunk of duplicate CSS in the English homepage bundle. Looking at the code, from what I can see this is only for a single button. Would hard-coding those styles in this (temporary) stylesheet be worth it to avoid the double-import?